[2024-11-19T01:05:06.344+0100] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T01:05:06.391+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load_data scheduled__2024-11-18T00:00:00+00:00 [queued]>
[2024-11-19T01:05:06.410+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load_data scheduled__2024-11-18T00:00:00+00:00 [queued]>
[2024-11-19T01:05:06.412+0100] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-19T01:05:06.446+0100] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_data> on 2024-11-18 00:00:00+00:00
[2024-11-19T01:05:06.464+0100] {standard_task_runner.py:72} INFO - Started process 210074 to run task
[2024-11-19T01:05:06.473+0100] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'load_data', 'scheduled__2024-11-18T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/raw_data_dag.py', '--cfg-path', '/tmp/tmp0yvc4jmx']
[2024-11-19T01:05:06.476+0100] {standard_task_runner.py:105} INFO - Job 20: Subtask load_data
[2024-11-19T01:05:07.307+0100] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.load_data scheduled__2024-11-18T00:00:00+00:00 [running]> on host zainab.
[2024-11-19T01:05:07.852+0100] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T00:00:00+00:00'
[2024-11-19T01:05:07.856+0100] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-19T01:05:07.859+0100] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-19T01:05:07.862+0100] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-11-19 00:05:06.394990+00:00
[2024-11-19T01:05:07.864+0100] {logging_mixin.py:190} INFO - Dag name:etl_pipeline and current dag run status:running
[2024-11-19T01:05:07.865+0100] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T01:05:07.898+0100] {base.py:84} INFO - Retrieving connection 'aws_default'
[2024-11-19T01:05:07.901+0100] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-11-19T01:05:50.568+0100] {hook.py:208} WARNING - Data lineage tracking is disabled. Register a hook lineage reader to start tracking hook lineage.
[2024-11-19T01:05:50.575+0100] {load.py:32} INFO - File successfully uploaded to S3: s3://zainycap-bucket/data/country_data_2024-11-18.parquet
[2024-11-19T01:05:50.584+0100] {python.py:240} INFO - Done. Returned value was: None
[2024-11-19T01:05:50.625+0100] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T01:05:50.630+0100] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load_data, run_id=scheduled__2024-11-18T00:00:00+00:00, execution_date=20241118T000000, start_date=20241119T000506, end_date=20241119T000550
[2024-11-19T01:05:50.690+0100] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-19T01:05:50.694+0100] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-19T01:05:50.698+0100] {logging_mixin.py:190} INFO - Dag name:etl_pipeline queued_at:2024-11-19 00:00:01.526754+00:00
[2024-11-19T01:05:50.701+0100] {logging_mixin.py:190} INFO - Task hostname:zainab. operator:PythonOperator
[2024-11-19T01:05:50.735+0100] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-19T01:05:50.765+0100] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T01:05:50.772+0100] {local_task_job_runner.py:245} INFO - ::endgroup::
